src/components/aws/services/awsSrv.js

```diff
@@ -0,0 +1,130 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra-web-app.aws').service('AwsSrv',
+        function (ENV, $log, $q, UtilitySrv, $window) {
+            'ngInject';
+
+         function _updateAwsConfig(config, options) {
+            options = options || {}; 
+            config = config || false;
+
+            if (!config) {
+               config = {
+                   region: options.region || ENV.s3Region || 'us-east-1',
+                   credentials: new AWS.CognitoIdentityCredentials({
+                       IdentityPoolId: options.IdentityPoolId || ENV.IdentityPoolId || 'us-east-1:d356a336-de8a-48c7-b67f-65c634462529'
```

Comment:
us-east-1:d356a336-de8a-48c7-b67f-65c634462529 
- does it has to be hard coded ?